reclist/reclist.py

```diff
@@ -1,8 +1,12 @@
 import collections
+import json
+
 from reclist.abstractions import RecList, rec_test
 from typing import List
 import random
 
+RESOURCE_METADATA_PATH = '/Users/piscoa01/Downloads/bbc-datalab-sounds-pesquet_dataset_2021-09_item_metadata_210930_all_items_enriched.ndjson'
```

Comment:
Could the resource metadata be auto-pulled with the rest of the data? I needed to manually copy and update the paths on my end to get this to work.

Reply:
That's a good point – I've changed that :)